docs-source/content/userguide/managing-domains/model-in-image/model-files.md

```diff
@@ -0,0 +1,156 @@
++++
+title = "Model files"
+date = 2020-03-11T16:45:16-05:00
+weight = 30
+pre = "<b> </b>"
+description = "Model file requirements, macros, and loading order."
++++
+
+#### Contents
+
+ - [Introduction](#introduction)
+ - [Sample model file](#sample-model-file)
+ - [Important notes about Model in Image model files](#important-notes-about-model-in-image-model-files)
+ - [Model file naming and loading order](#model-file-naming-and-loading-order)
+ - [Model file macros](#model-file-macros)
+   - [Using secrets in model files](#using-secrets-in-model-files)
+   - [Using environment variables in model files](#using-environment-variables-in-model-files)
+   - [Combining secrets and environment variables in model files](#combining-secrets-and-environment-variables-in-model-files)
+
+#### Introduction
+
+This document describes basic Model in Image model file syntax, naming, and macros. For additional information, see the [WebLogic Deploy Tool](https://github.com/oracle/weblogic-deploy-tooling) documentation.
+
+{{% notice tip %}} The WDT 'discover tool' is particularly useful for generating model files from an existing domain home.
+{{% /notice %}}
+
+#### Sample model file
+
+Here's an example of a model `.yaml` file that defines a WebLogic Administration Server and dynamic cluster.
```

Comment:
so i read this, and it makes no sense to me - what is this file for? what are all these things in here? there is no explanation of what this is

Reply:
@markxnelson As we discussed in slack, let's follow up via a zoom discussion when you have a chance.